### PR TITLE
Add documentation for the new Arve integration

### PR DESCRIPTION
**Pull request by ikalnyi**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/113156
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/5285
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/arve.markdown [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  - sensor
+ha_integration_type: hub
+ha_external_link: https://github.com/home-assistant/core/tree/dev/homeassistant/components/arve
+---
+
+The **Arve** {% term integration %} provides the ability to monitor real life air quality data, measured by your [Arve](https://www.arveair.com) device.
+
+{% include integrations/config_flow.md %}
+
+To create an integration for an Arve device, the user needs to fill in two required fields.
```
Comment (c0ffeeca7) [inline source/_integrations/arve.markdown]:
```suggestion
To create an integration for an Arve device, you need to fill in two required fields.
```

File: source/_integrations/arve.markdown [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+Those are _Home Assistant acess token_ and _Arve customer token_.
+_Home Assistant access token_ and _Arve customer token_ can be obtained by the user on the [Arve web platform](https://dashboard.arveair.com).
+
+## Measurement entities
+
+At the moment, Arve device has following measurement entities:
```
Comment (c0ffeeca7) [inline source/_integrations/arve.markdown]:
```suggestion
At the moment, an Arve device has the following measurement entities:
```

File: source/_integrations/arve.markdown [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+AQI - air quality index;
+
+CO2 - co2 measurement in parts per million;
+
+Humidity - humidity percentage;
+
+PM10 - concentration of particulates with diameter less than 10µm;
+
+PM25 - concentration of particulates with diameter less than 2.5µm;
```
Comment (c0ffeeca7) [inline source/_integrations/arve.markdown]:
```suggestion
PM10 - concentration of particulates with a diameter less than 10&nbsp;µm;

PM25 - concentration of particulates with a diameter less than 2.5&nbsp;µm;
```

File: source/_integrations/arve.markdown [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+Temperature - temperature measurement in celsius degrees;
```
Comment (c0ffeeca7) [inline source/_integrations/arve.markdown]:
```suggestion
Temperature - temperature measurement in degrees Celsius;
```

File: Gemfile.lock [hunk 5]
```diff
@@ -127,6 +127,8 @@ GEM
       google-protobuf (>= 3.25, < 5.0)
     sass-globbing (1.1.5)
       sass (>= 3.1)
+    sassc (2.1.0)
```
Comment (c0ffeeca7) [inline Gemfile.lock]: The Gemfile changes are unrelated and should not be part of this PR.

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @ikalnyi :+1:

**Comment by ikalnyi** [comment]
Thank you for your help, @c0ffeeca7

File: source/_integrations/arve.markdown [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+ha_codeowners:
+  - "@ikalnyi"
+ha_domain: arve
+ha_platforms:
+  - sensor
+ha_integration_type: hub
+ha_external_link: https://github.com/home-assistant/core/tree/dev/homeassistant/components/arve
+---
+
+The **Arve** {% term integration %} provides the ability to monitor real life air quality data, measured by your [Arve](https://www.arveair.com) device.
```
Comment (c0ffeeca7) [inline source/_integrations/arve.markdown]:
```suggestion
The **Arve** {% term integration %} provides the ability to monitor real-life air quality data, as measured by your [Arve](https://www.arveair.com) device.
```

**Comment by ikalnyi** [comment]
Is there anything that I should or can do about `netlify` check failures ?

**Comment by c0ffeeca7** [comment]
> Is there anything that I should or can do about `netlify` check failures ?

No. I need help from frenck on this. He will take a look.

**Comment by ikalnyi** [comment]
> > Is there anything that I should or can do about `netlify` check failures ?
> 
> No. I need help from frenck on this. He will take a look.

Understood, thank you

**Comment by ikalnyi** [comment]
Hello, is there any updates on the problem ?

**Comment by ikalnyi** [comment]
Hello @c0ffeeca7, @frenck, @joostlek, will it be helpful if I just open a new pull request ? I think that when I accidentally added `Gemfile.lock` and then deleted it, something broke.

**Comment by joostlek** [comment]
I don't think that should matter tbf, you can maybe try squashing all commits and pushing it as one

**Comment by joostlek** [comment]
(Oh and don't worry, we marked this PR as parent-merged, so it will be fixed before the release eventually :) )

**Comment by ikalnyi** [comment]
> (Oh and don't worry, we marked this PR as parent-merged, so it will be fixed before the release eventually :) )

Thanks for the information

**Comment by frenck** [comment]
superseded by #32532 (which fixes CI)
